tools/wptrunner/wptrunner/tests/browsers/test_firefox.py

```diff
@@ -0,0 +1,30 @@
+import sys
+from os.path import join, dirname
+
+import pytest
+
+from mozlog import structured
+
+sys.path.insert(0, join(dirname(__file__), "..", "..", ".."))
+
+firefox = pytest.importorskip("wptrunner.browsers.firefox")
+
+logger = structured.structuredlog.StructuredLogger("TestBrowsersFirefox")
+structured.set_default_logger(logger)
+
+
+def test_server_location():
+    browser = firefox.FirefoxBrowser(logger,
+                                     "foo/firefox",
+                                      "foo/prefs",
+                                      "testharness.js",
+                                      config={
+                                          "ports": {"http": [1234]},
```

Comment:
This test doesn't seem that useful. It ensures that you write something to some file, but it doesn't really check that the contents are right. It seems more useful to set things up so you can check you get the exact content you expect compared against the file you removed. Obviously this would then have to be changed when new servers are added or removed, but it would have the advantage of actually checking that something functional is produced.